spec/factories/passengers.rb

```diff
@@ -56,5 +56,10 @@
     trait :permanent do
       permanent { true }
     end
+
+    trait :subscribed_to_sms do
+      subscribed_to_sms { true }
+      association :carrier
+    end
```

Comment:
Now you aren't using this trait any more. I don't think it's useful to have as a trait tbh

Reply:
Oh, I forgot I made that.